### PR TITLE
fix(chat): Download/invite popup small issues

### LINKED ISSUE #11926: [chat] Download/invite popup small issues
# Bug Report

image

- the popup is not centered
- the link (status.im) is wrong and should be probably clickable

### Additional Information

- Status desktop version: master
- Operating System: linux

### PR DESCRIPTION
**Pull request by caybro**
- make the popup centered across the whole window
- fix (and extract) the get/download link to `Constants`
- fix some memleaks

Fixes #11926

### Affected areas

InviteFriendsPopup

### Screenshot of functionality (including design for comparison)

- [x] I've checked the design and this PR matches it

image

### CODE REVIEW

File: ui/imports/shared/views/NoFriendsRectangle.qml [hunk 1]
```diff
@@ -41,6 +41,7 @@ Item {
         id: inviteFriendsPopup
         InviteFriendsPopup {
             rootStore: noContactsRect.rootStore
+            onClosed: destroy()
```
Comment (micieslak) [inline ui/imports/shared/views/NoFriendsRectangle.qml:44]: `destroyOnClose: true` ?

File: ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml [hunk 2]
```diff
@@ -81,7 +81,7 @@ Item {
                     parent.font.underline = false
                 }
                 onClicked: {
-                    inviteFriendsPopup.open();
+                    Global.openPopup(inviteFriendsPopup)
```
Comment (micieslak) [inline ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml:84]: if used via Global.openPopup, maybe it's good moment to move `inviteFriendsPopup` component to `Popups`?

File: ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml [hunk 3]
```diff
@@ -95,8 +95,11 @@ Item {
         anchors.top: links.bottom
     }
 
-    InviteFriendsPopup {
+    Component {
         id: inviteFriendsPopup
-        rootStore: element.rootStore
+        InviteFriendsPopup {
+            rootStore: element.rootStore
+            onClosed: destroy()
```
Comment (micieslak) [inline ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml:102]: we have `destroyOnClose: true`

**Review by micieslak — COMMENTED** [review]
Some minor comments

File: ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml (hunk 2 above)
Comment (caybro) [reply ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml:84]: It's used only from here afaics

File: ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml (hunk 3 above)
Comment (caybro) [reply ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml:102]: Yeah, why not; same thing :)

File: ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml (hunk 2 above)
Comment (caybro) [reply ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml:84]: Yeah, can do

File: ui/imports/shared/views/NoFriendsRectangle.qml (hunk 1 above)
Comment (caybro) [reply ui/imports/shared/views/NoFriendsRectangle.qml]: Done

File: ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml (hunk 3 above)
Comment (caybro) [reply ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml]: Done

File: ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml (hunk 2 above)
Comment (caybro) [reply ui/app/AppLayouts/Chat/panels/EmptyChatPanel.qml:84]: On a second thought, I decided not to... I'd rather avoid these refactors in a simple bugfix PR like this

**Review by micieslak — APPROVED** [review]

**Review by alexandraB99 — APPROVED** [review]

**Pull request merged**